storage/reads/table.go

```diff
@@ -127,6 +127,7 @@ func (t *table) appendTags() {
 		if v != nil {
 			b := arrow.NewStringBuilder(t.alloc)
 			b.Reserve(t.l)
+			b.ReserveData(t.l*len(v))
```

Comment:
Why a call to both `Reserve` and `ReserveData`?

Reply:
The call to `Reserve` is for the number of entries and the second is for the binary data that will be appended so they're different numbers.